notebooks/week5/01_experiment_with_new_data.py

```
@@ -0,0 +1,68 @@
+    Wind_Speed,
+    general_diffuse_flows,
+    diffuse_flows,
+    DayOfWeek_1,
+    DayOfWeek_2,
+    DayOfWeek_3,
+    DayOfWeek_4,
+    DayOfWeek_5,
+    DayOfWeek_6,
+    IsWeekend_1
+FROM new_test
+""")
```

Ruff (0.7.0)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)